crates/devolutions-pedm/src/utils.rs

```diff
@@ -115,6 +116,7 @@ pub(crate) fn file_hash(path: &Path) -> anyhow::Result<Hash> {
 
 pub(crate) fn ensure_protected_directory(dir: &Path, _readers: Vec<Sid>) -> anyhow::Result<()> {
     // FIXME: Underlying behaviour of security primitives must be corrected before this can work
```

Comment:
question: Is it something important we need to uncomment for the V1?

Reply:
Ideally we would uncomment this. I will try it on my side as this was a source of issues.

We are setting directory permissions in the installer, but I'd consider it safer to also use this function and enforce permissions on access.